frontend/src/scenes/experiments/ExperimentView/DataCollectionCalculator.tsx

```diff
@@ -136,9 +136,29 @@ export function DataCollectionCalculator({ experimentId }: ExperimentCalculatorP
             <div className="w-full">
                 <div className="mb-4 experiment-preview-row">
                     <div className="flex items-center">
-                        <b>Minimum acceptable improvement</b>
-                        <Tooltip title="Minimum acceptable improvement is a calculation that estimates the smallest significant improvement you are willing to accept.">
-                            <IconInfo className="ml-1 text-muted text-xl" />
+                        <b>Minimum detectable effect</b>
```

Comment:
Should [this tooltip](https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/experiments/ExperimentView/DataCollection.tsx#L42) also use `Minimum detectable effect` instead of `Minimum Acceptable Improvement`, or are they different things?

Reply:
good catch :)